src/Aquarius.ONE.ClientSDK/ClientSDK.cs

```diff
@@ -34,6 +34,23 @@ public ClientSDK()
             Logger = new EventLogger();
             InstantiateAPIs();
         }
+
+        public ClientSDK(string environment)

```

Comment:
Re-iterating the comment that @AQI-DanG  made... perhaps use Enum environment instead of string

Reply:
Updated to use the enum, also added another constructor that mimics the existing one but uses the enum instead of the string.